exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+
+In the experiment, the blade tip passes near the wind-tunnel walls at peak deflection.  Since the Nalu-Wind domain extends several chord lengths away from the wing, the AMR-Wind domain was chosen to be somewhat larger than the wind tunnel in order to avoid collisions of the overset boundary with the slip wall at peak deflection, and additional padding was provided to ensure the results are not contaminated by the effect of the artificial slip wall boundaries.  Future studies are planned to investigate the impact of the choice of AMR-Wind domain size on the simulation results.
+
+**Grid Generation**
+
+<img src="figures/mesh_overview_100.png" border="0" alt=""/>
+
+The Nalu-Wind grid was generated using Pointwise, and a cut-away view is shown in the left figure above.  The wing tip surface grid used an O-grid surrounding an H-grid, to which elliptic smoothing was applied.  The outer boundary of this wing tip grid was extruded to yield the complete surface mesh, shown in red above.  Finally, the volume mesh was created through hyperbolic extrusion of the surface mesh.  The grid for AMR-Wind was generated using its internal grid generator, and has one refinement region in the interior, centered around the wing.  This grid is shown in the right figure above.  Metrics for these grids are given in the table below.
+
+| Nalu-Wind Grid Property         | Value                     |   | AMR-Wind Grid Property   | Value                  |
+| :---                            | ---                       | - | :---                     | ---                    |
+| Number of cells                 | 8,736,600                 |   | Domain size              | 3 m x 3 m x 1.5 m      |
+| First cell height               | 1.0E-06 m                 |   | Level 0 dimensions       | 128 x 128 x 64         |
+| First cell y+                   | 0.2                       |   | Number of cells          | 5,767,168              |
+| Cells around airfoil            | 500                       |   | Level 0 spacing          | ~0.024 m               |
+| Number of spanwise layers       | 100                       |   | Level 1 spacing          | ~0.012 m               |
+
+**Setup**
+
+1. Clone the Exawind-Benchmarks repository by using the following syntax:
+
+	```bash
+	$ git clone --recursive git@github.com:Exawind/exawind-benchmarks.git
+	```
+2. Once the repository has been cloned, navigate to the Pazy-Wing directory:
+
+	```bash
+   $ cd exawind-benchmarks/exawind/Pazy_Wing
+   ```
+3. Modify the "loadmod.sh" script to point to your ExaWind installation.  We assume here that you are using Exawind-Manager.
```

Comment:
Add a link here to https://github.com/Exawind/exawind-manager just in case people don't know what that is.

Reply:
Done!